crdppf/static/js/Crdppf/map.js

```diff
@@ -70,15 +70,6 @@ Crdppf.Map.prototype = {
             }
         });
 
-        control.events.register("beforefeatureselected", this, function(e) {
-            Crdppf.Map.selectLayer.removeAllFeatures();
-            Crdppf.currentProperty = null;
-            var propertySelectionWindow = Ext.getCmp('propertySelectionWindow');
-            if (propertySelectionWindow) {
-                propertySelectionWindow.destroy();
-            }
-        });
-
         // define actions on feature selection
```

Comment:
it existed twice in the code (ctrl-D). so it's stille one copie left :) (line 64)